code/addons/test/package.json

```diff
@@ -151,11 +152,41 @@
     ],
     "nodeEntries": [
       "./src/preset.ts",
-      "./src/vitest-plugin/index.ts",
-      "./src/vitest-plugin/global-setup.ts",
       "./src/postinstall.ts",
-      "./src/node/vitest.ts",
-      "./src/node/coverage-reporter.ts"
+      {
+        "file": "./src/dummy.ts",
+        "formats": [
+          "esm"
+        ]
+      },
+      {
+        "file": "./src/node/vitest.ts",
+        "formats": [
+          "esm",
+          "cjs"
+        ]
+      },
+      {
+        "file": "./src/node/coverage-reporter.ts",
+        "formats": [
+          "esm",
+          "cjs"
+        ]
+      },
+      {
+        "file": "./src/vitest-plugin/index.ts",
+        "formats": [
+          "cjs",
+          "esm"
+        ]
+      },
+      {
+        "file": "./src/vitest-plugin/global-setup.ts",
+        "formats": [
+          "cjs",
+          "esm"
+        ]
+      }
```

Comment:
are you sure all of these needs _both_ formats? Instinctively I'd think only the ESM format is necessary, or is it because it depends on the project's `"type": "module"`?

Reply:
That's correct!